test/e2e/kubevirt.go

```diff
@@ -1643,24 +1770,29 @@ runcmd:
 				WithPolling(time.Second).
 				Should(Succeed(), step)
 
-			step = by(vmi.Name, "Wait for cloud init to finish at first boot")
-			output, err := kubevirt.RunCommand(vmi, "cloud-init status --wait", time.Minute)
-			Expect(err).NotTo(HaveOccurred(), step+": "+output)
-
 			// expect 2 addresses on dual-stack deployments; 1 on single-stack
 			step = by(vmi.Name, "Wait for addresses at the virtual machine")
-			expectedNumberOfAddresses := len(strings.Split(netConfig.cidr, ","))
+			expectedNumberOfAddresses := len(cidrs)
 			expectedAddreses := virtualMachineAddressesFromStatus(vmi, expectedNumberOfAddresses)
 			expectedAddresesAtGuest := expectedAddreses
-			testPodsIPs := podsMultusNetworkIPs(iperfServerTestPods, podNetworkStatusByNetConfigPredicate(netConfig))
-
-			step = by(vmi.Name, "Expose VM iperf server as a service")
-			svc, err := fr.ClientSet.CoreV1().Services(namespace).Create(context.TODO(), composeService("iperf3-vm-server", vmi.Name, 5201), metav1.CreateOptions{})
-			Expect(svc.Spec.Ports[0].NodePort).NotTo(Equal(0), step)
+			testPodsIPs := podsMultusNetworkIPs(iperfServerTestPods, podNetworkStatusByNetConfigPredicate(namespace, cudn.Name, strings.ToLower(string(td.role))))
+
+			serverPort := int32(5201)
```

Comment:
I *really* dislike this hard-coded port number. 

At the very least, let's make the issue more prominent by specifying it as a const and re-using it.

But I'm hoping to see a random port number.

Reply:
I can use a random number